app/views/insured/plan_shoppings/_plan_filters.html.erb

```diff
@@ -53,12 +53,12 @@
             <div class="modal-footer" style="padding:10px">
               <center>
               <button type="button" class="btn btn-default btn-default" data-dismiss="modal">Cancel</button>
-              <%=link_to l10n("confirm") , 'https://dc.checkbookhealth.org/sadpi/dc/2019/', class: "btn btn-primary" %>
+              <%= sanitize link_to sanitize(l10n("proceed")) , sanitize(l10n("checkbook_sadpi_health_url")), class: "btn btn-primary" %>
```

Comment:
why this line doesn't have only one sanitize like line 35 ?

Reply:
I have updated the code.